RELEASE.md

```diff
@@ -1,3 +1,11 @@
 # Release Instructions
 
+The following commands should be run when doing a release, where `{version}` is `major`, `minor`, or `patch`.
+
+```sh
+bin/release {version}
+git push
+npm publish
+```
```

Comment:
@taylorotwell I'm not sure if this is totally true. We may also want to consolidate this all into the `release` script.

Let me know.

Reply:
Does `{version}` needs to be `v1.0.0` or `1.0.0`?

Reply:
> The following commands should be run when doing a release, where `{version}` is `major`, `minor`, or `patch`.